assets/build/webpack.config.watch.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
       target: config.devUrl,
       proxyUrl: config.proxyUrl,
       watch: config.watch,
+      delay: 500,
```

Comment:
In the case you're editing a file that causes a refresh and you save the file multiple times, there's the possibility that you get an outdated version if the browser is in the middle of a refresh when the save is detected.

I ran into this and used [`reloadDebounce`](https://browsersync.io/docs/options/#option-reloadDebounce) instead. Seems pointless to trigger multiple refreshes in a row anyways.